components/common-go/grpc/grpc.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package grpc
+
+import (
+	"time"
+
+	grpc_middleware "github.com/grpc-ecosystem/go-grpc-middleware"
+	grpc_opentracing "github.com/grpc-ecosystem/go-grpc-middleware/tracing/opentracing"
+	"github.com/opentracing/opentracing-go"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/backoff"
+	"google.golang.org/grpc/keepalive"
+)
+
+// maxMsgSize use 16MB as the default message size limit.
+// grpc library default is 4MB
+const maxMsgSize = 1024 * 1024 * 16
```

Comment:
The main reason to adjust this value is related to some observed metrics that could indicate the current value is low.
`grpc_server_handled_total{grpc_code="ResourceExhausted",grpc_method="StartWorkspace",grpc_service="wsman.WorkspaceManager",grpc_type="unary"} 8
`